api/src/opentrons/protocol_engine/errors/exceptions.py

```diff
@@ -199,3 +199,11 @@ class LabwareMovementNotAllowedError(ProtocolEngineError):
 
 class LocationIsOccupiedError(ProtocolEngineError):
     """An error raised when attempting to place labware in a non-empty location."""
+
+
+class NonExistentLabwareError(ProtocolEngineError):
```

Comment:
niptick can we change this to:
```suggestion
class LabwareDoesNotExistError(ProtocolEngineError):
```

Reply:
Agreed. `...DoesNotExist` is consistent with `LabwareDefinitionDoesNotExistError` and `LabwareOffsetDoesNotExistError`.

Reply:
However:

* I don't think it's strictly correct to call this a `Labware`. A `Labware`, to me, means a specific loaded instance, as opposed to a labware definition. The new search function added in this PR searches through a set of labware definitions, not loaded instances.
* We already have `LabwareDefinitionDoesNotExistError`. `loadLabware` raises it when it can't find anything with the provided `loadName`+`namespace`+`version`. Should we just reuse that?